pom.xml

```diff
@@ -199,6 +199,8 @@
     </dependencies>
 
     <build>
+      <defaultGoal>clean package</defaultGoal>
+      <finalName>${project.name}</finalName>
```

Comment:
Instead of adding a new property here, you can instead change the one near the top of the file that's just for this:
```xml
<project.finalName>${project.outputName}-${project.version}</project.finalName>
```